src/main.c

```diff
@@ -768,8 +756,6 @@ static gchar* info_formatter_json_base(RaucManifest *manifest, gboolean pretty)
 	str = json_generator_to_data(gen, NULL);
 
 	json_node_free(root);
```

Comment:
Without looking into the json-api: Should the root pointer still be freed when declared as autoptr?

Reply:
Absolutely not, thanks for catching this!
json_node_free is the autoptr cleanup function registered for the JsonNode type, so I have to remove this as well.